docs/technical/feature-flagging.md

```diff
@@ -49,6 +49,9 @@ feature once things are stabilized.
 
 ## How to test
 
-To opt-in for testing preview features, set the
-`GITHUB_DESKTOP_PREVIEW_FEATURES` environment variable to any value and launch
-the Desktop app.
+Set the `GITHUB_DESKTOP_PREVIEW_FEATURES` environment variable to any value to opt-in for preview featurs.
+
+Don't have that environment variable? 
+No worries, simply create it. (here's a [handy guide](https://www.schrodinger.com/kb/1842) for doing that on most major OSs).
+
+Note, if you would like to opt-out of preview feautures, you will need to remove this variable from your environment.
```

Comment:
Do you have to restart Desktop each time you change the setting?

Reply:
Good point. We should make that clear.